google-cloud-tools-plugin/src/com/google/gct/login/IntelliJGoogleLoginMessageExtender.java

```diff
@@ -0,0 +1,11 @@
+package com.google.gct.login;
+
+import com.google.gct.idea.util.GctBundle;
+
+public class IntelliJGoogleLoginMessageExtender implements GoogleLoginMessageExtender {
+
+  @Override
+  public String additionalLogoutMessage() {
+    return GctBundle.message("clouddebug.logout.additional.message");
```

Comment:
Is this only activated if we have debugger sessions active? It seems odd that all users will see this message?